src/Common/DataObjects/Pitch.ts

```diff
@@ -363,6 +377,25 @@ export class Pitch {
         return !(p1 === p2);
     }
 
+    //These don't take into account accidentals! which isn't needed for our current purpose
+    public OperatorGreaterThan(p2: Pitch): boolean {
```

Comment:
To make it more clear i would call these OperatorGreaterFundamentalThan, OperatorLessFundamentalThan